ducktape/command_line/main.py

```diff
@@ -158,18 +159,19 @@ def main():
 
     # Initializing the cluster is slow, so do so only if
     # tests are sure to be run
+    cluster = None
```

Comment:
Why this assignment? Python isn't scoped the same way as other languages, and if an exception occurs you're going to exit anyway.